docs/cypress/e2e/links.cy.ts

```diff
@@ -10,44 +10,60 @@ type EvtName =
   | 'REQUESTING'
   | 'RETURNING'
   | 'NO_HREF';
-let allLinks: string[] = [];
-const numberOfLinks = 121;
-const requestedLinks: Set<string> = new Set();
 
-before(() => {
-  cy.task('readSitemapLinks').then((links: string[]) => {
-    allLinks = allLinks.concat(links);
-  });
-});
+describe(`All pages on Sitemap`, () => {
+  let allLinks = [];
+  const requestedLinks: Set<string> = new Set();
 
-describe('Local Sitemap', () => {
-  it('should have 119 links', () => {
-    expect(allLinks.length).to.eq(numberOfLinks);
+  before(() => {
+    cy.request('sitemap.xml').then((response) => {
+      allLinks = Cypress.$(response.body)
+        .find('loc')
+        .toArray()
+        .map((el) => el.innerText)
+        .map((link) =>
+          link
+            .replace(`${BASE_URL}/`, '')
+            .replace('https://www.dev.ui.docs.amplify.aws/', '')
+            .replace('https://ui.docs.amplify.aws/', '')
+        );
+    });
   });
-});
 
-describe(`All pages on Sitemap`, () => {
-  it.each(numberOfLinks)('all links on page %k should work', (i: string) => {
-    const link = allLinks[i];
-    cy.task('log', `🧪[TESTING...] page ${BASE_URL}/${link}`);
-    cy.visit(link || '/');
-    cy.get('a').each((el) => hrefWorks(el, link));
+  it('should succesfully load each url in the sitemap', () => {
+    allLinks.forEach((link, idx) => {
+      cy.task('log', `🧪[TESTING...] page #${idx} ${BASE_URL}/${link}`);
+      cy.visit(link || '/');
```

Comment:
Can we add a query string to easily identify and filter these requests in our merics?
```
cy.visit(URL, qs: {'cypress': true}) 
```

same comment for the `cy.request` call bellow.

Reply:
Good idea. Done.